src/content/Blog/running-vllm-on-akash/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Running vLLM on Akash"
+description: Learn how to save money by replacing Closed AI models like GPT-4 by running open source LLM's on Akash with vLLM. vLLM is the most powerful open source LLM server for multiuser / high concurrent request LLM services. We show you how to run fully featured open source llm application stack on Akash including with OpenWeb UI Chat application and AI Agent workflows for using LangChain and Crew AI. Use this deployment to replace GPT-4 for your entire office or team.  
+
+pubDate: "2024-5-15"
+draft: false
+showcase: true 
+featured: true
+
+categories:
+  - AI & ML
+  - Insights
+  - Updates
+  - News
+
+tags:
+  - AI & ML
+  - Insights
+  - Updates
+  - News
+
+githubLink: "https://github.com/akash-netowork/awesome-akash/vllm" # <--- This is the github link of the project
+
+contributors:
+  - Logan Cerkovnik
+
+bannerImage: ./banner-image.jpg
+---
+# Running vLLM on Akash
+By [Logan Cerkovnik](https://www.twitter.com/ThumperAI) & [Anil Murty](https://twitter.com/_Anil_Murty_)
+
+[![Running vLLM on Akash](https://res.cloudinary.com/marcomontalbano/image/upload/v1715742467/video_to_markdown/images/youtube--_hv9681CV0Y-c05b58ac6eb4c4700831b2b3070cd403.jpg)](https://youtu.be/_hv9681CV0Y "Running vLLM on Akash")
```

Comment:
Let's embed the youtube video directly below the first paragraph (above "Delving into vLLM", the header image will already display at the top of the post automatically

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/_hv9681CV0Y?si=-h_-HgfcWC-_JoPD" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```